app/assets/javascripts/app/views/conversations_form_view.js

```diff
@@ -1,52 +1,164 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
 app.views.ConversationsForm = Backbone.View.extend({
+  el: ".conversations-form-container",
 
   events: {
-    "keydown textarea#conversation_text" : "keyDown",
+    "keydown textarea.conversation-message-text": "keyDown",
+    "submit #conversation-new": "onSubmitForm"
   },
 
   initialize: function(opts) {
-    this.contacts = _.has(opts, "contacts") ? opts.contacts : null;
-    if(!this.contacts || this.contacts.length === 0) {
-      this.displayNoContactsMessage();
-      return;
+    this.conversationParticipants = [];
+
+    this.search = new app.views.SearchBase({
+      el: this.$el.find("#new-conversation"),
+      typeaheadInput: this.getTypeaheadElement(),
+      customSearch: true,
+      autoselect: true,
+      remoteRoute: "/contacts"
```

Comment:
Hmm... Right. So, should I create a new route or just use a parameter in `ContactsController#index`?